src/main/resources/templates/layouts/main.html

```diff
@@ -17,9 +17,12 @@
 <body id="app">
 
 <nav id="page-navbar" class="navbar">
-    <a href="" data-th-href="@{/}" class="page-navbar__item page-navbar__item-selected">
+    <a href="" data-th-href="@{/}" class="page-navbar__item" th:classappend="${#request.requestURI == '/probes' ? 'page-navbar__item-selected' : ''}">
```

Comment:
Never ever use `==` to compare strings.

```suggestion
    <a href="" data-th-href="@{/}" class="page-navbar__item" data-th-classappend="${#request.requestURI.equals('/probes') ? 'page-navbar__item-selected' : ''}">
```

Reply:
Sorry for that, I'll keep that in mind.